editor/plugins/animation_tree_editor_plugin.cpp

```diff
@@ -127,6 +128,10 @@ void AnimationTreeEditor::edit_path(const Vector<String> &p_path) {
 	} else {
 		current_root = ObjectID();
 		edited_path = button_path;
+
+		for (int i = 0; i < editors.size(); i++) {
+			editors[i]->hide();
```

Comment:
I suggest calling `editors[i]->edit(Ref<AnimationNode>());` before `editors[i]->hide();`.

The commit as is fixes the bug. However, when the node is removed, the editor is left holding a reference to an invalid object. Which is not a problem now, but could be in the future.

I'm not saying the commit is wrong. I'm saying there is an opportunity to prevent a possible future problem here.